lib/trie/trie.go

```diff
@@ -1,19 +1,20 @@
 package trie
 
 import (
-	sebakcommon "boscoin.io/sebak/lib/common"
-	"github.com/ethereum/go-ethereum/common"
+	gocommon "github.com/ethereum/go-ethereum/common"
```

Comment:
Sorry to nitpick, but I'd prefer something that reference Ethereum. What about `ethcommon` ?

Reply:
@Geod24 Oh, that's better :)